security-privacy-questionnaire.md

```diff
@@ -0,0 +1,73 @@
+
+### 2.3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+
+No data is exposed without the user explicitly choosing what files or directories to expose to the web site.
+
+### 2.4. How does this specification deal with sensitive information?
+
+No data is exposed without the user explicitly choosing what files or directories to expose to the web site, so only sensitive data that the user explicitly decides to share via this API will be shared. Furthermore, this API is only exposed in secure contexts, and third-party iframes (i.e. iframes that are cross origin from the top-level frame) won't be able to show pickers or permission prompts and can only access data they were already granted access to from a top-level same origin frame.
+
+### 2.5. Does this specification introduce new state for an origin that persists across browsing sessions?
+
+Yes, this specification lets websites store handles they've gotten access to (via a file or directory picker) in IndexedDB. User agents could also persist the permission grants that go with these handles, but at least in the Chrome implementation, these permission grants will only be persistent for installed PWAs. The drive-by web will only have enough state to allow it to re-prompt for access, but the access itself won't be persistent.
+
+Furthermore, the user will be able to clear storage (storage is just IndexedDB) and/or revoke permissions to clear the state that was persisted, similarly to how other permissions work.
+
+Websites can also store any state they like in files they get write access to via this API. This state would not be cleared when clearing browser data, however access to this state would be removed. If a user later picks the same files or directories again to give the website access to them, the websites will regain access to whatever state they persisted.
+
+### 2.6. What information from the underlying platform, e.g. configuration data, is exposed by this specification to an origin?
+
+Anything that exists on disk in files could be exposed by the user to the web. However, user agents are encouraged to maintain a block list of certain directories with particularly sensitive files, and thus somewhat restrict which files and directories the user is allowed to select. For example, things like Chrome's "Profile" directory, and other platform configuration data directories are likely going to be on this block list.
+
+### 2.7. Does this specification allow an origin access to sensors on a user’s device
+
+No (unless a device exposes such sensors as files or directories).
```

Comment:
Would it be worth adding a sentence clarifying that we recommend directories exposing devices, like `/dev`, to be included in the block list?